FIRMWARE

```diff
@@ -7,35 +7,36 @@ Hash: SHA256
 # ckb-next does NOT read this file when flashing firmware manually. It will allow you to load any valid FW blob (at your own risk).
 #
 # -------->>>>>> for Scimitar is a 2.4 available, for Strafe 2.05 !
-# -------->>>>>> What code is valid for K70 (non RGB)? Lastz code at corsair is 2.04
 #
 
 !BEGIN FW ENTRIES
 
-# Vendor  Model       FW   URL                                                 Min.ckb File Name              SHA256
-  Corsair K70RGB      2.05 http://www3.corsair.com/software/HID/K70RGB.zip     0.2.7   K70RGB_APP_V205.bin    3e43bdcc5077dc413fbdee7ff6e57a978f758599142eef6979a96e3b8c3a566a
-  Corsair K70LUX      2.04 http://www3.corsair.com/software/HID/K70LUX.zip     0.2.7   K70LUX_APP_V204.bin    c2b5411e1dce391788eca294563801b32a14bf7802eb29688213f8f07e112966
-  Corsair K95RGB      2.05 http://www3.corsair.com/software/HID/K95RGB.zip     0.2.7   K95RGB_APP_V205.bin    8d0680dd033d6020a53bc9bfa83e761143699d1122f1d0133e95ad3056d63a15
-  Corsair STRAFE      2.04 http://www3.corsair.com/software/HID/STRAFE.zip     0.2.7   STRAFE_APP_V204.bin    51156ae7aa5b04405668582496cf28bbfb2e492accf93c65c3072aca9ef7ed65
-  Corsair STRAFERGB   2.05 http://www3.corsair.com/software/HID/STRAFERGB.zip  0.2.7   STRAFERGB_APP_V205.bin dff16fec9a0ae0d33aad0482f9ea02e3e55e231706b8c4e7fb66820aba98adc5
-  Corsair M65RGB      2.04 http://www3.corsair.com/software/HID/M65RGB.zip     0.2.7   M65RGB_App_v204.bin    e1f09ce462f77071f91d92951410c14fb18061dd9760b3de89e45d1ca8b6b46c
-  Corsair SCIMITARRGB 2.03 http://www3.corsair.com/software/HID/Scimitar.zip   0.2.7   Scimitar_App_v203.bin  59194c3ae8a6ae0a801f7b6cf528ad292ae996923699cf063c7deed38fb326aa
+# Vendor  Model         FW   URL                                                    Min.ckb File Name                   SHA256
+  Corsair K70RGB        2.05 https://www3.corsair.com/software/HID/K70RGB.zip       0.2.7   K70RGB_APP_V205.bin         3e43bdcc5077dc413fbdee7ff6e57a978f758599142eef6979a96e3b8c3a566a
+  Corsair K70LUX        2.04 https://www3.corsair.com/software/HID/K70LUX.zip       0.2.7   K70LUX_APP_V204.bin         c2b5411e1dce391788eca294563801b32a14bf7802eb29688213f8f07e112966
+  Corsair K70LUXRGB     2.05 https://www3.corsair.com/software/HID/K70LUXRGB.zip    0.2.7   K70LUX_APP_V205.bin         a1a0a4b2f74890eb708435b21ab197a301ea668269bca895f89f7301c354e83d
+  Corsair K70RAPIDFIRE  2.05 https://www3.corsair.com/software/HID/K70RAPIDFIRE.zip 0.2.7   K70RAPIDFIRE_APP_V205.bin   6b1d2bb962ffd987ce5eb25c46b02a9069c5f79ca05d7b2cc71c749fe9813536
```

Comment:
Okay, you can probably just comment it out for now. On irc I proposed creating a separate issue that would track the current support status of different keyboards (some of them are fully supported (I suppose), some of them are supported in an add-and-pray style with just the array of models being updated) just to keep track of everything (we already have k65-related info "somewhere"). Because I didn't follow the project for these two years I don't know what level of support is provided for the keyboards listed in Readme, I can only tell that my own keyboard (Strafe RGB) seems to be fully supported.